### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by nicolasmarino99**

### CODE REVIEW

**Comment by Torres-ssf** [comment]
### Pull Request Under Review By Sérgio (TSE)

**Comment by Torres-ssf** [comment]
### Great job @nicolasmarino99 :clap:
You did nice work on this project, but we need to do some changes to make it in line with Microverse guidelines.

- [x] Its a good practice to use the git-flow methodology. Check this [link](https://guides.github.com/introduction/flow/) You can try it on future projects. It's very easy and simple and will make you look like a PRO dev :thumbsup:

- [x] Try to make the footer match the same style from the original page

- [x] Your footer must contain the NW logo.

- [x] Microverse guidelines require that your page must be responsive in smaller screens sizes. 


Screenshot from 2019-08-05 21-32-36

Screenshot from 2019-08-05 21-34-51

Screenshot from 2019-08-05 21-37-11

Please, kindly make these changes and submit for code review again. Happy coding

File: index.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                        </g>
+                        </g>
+                    </svg>
+                </a>
+            </div>
+        </nav>
+        <div class="row header-bottom">
+        </div>
+        <div class="w-100"></div>
+        <div class="row" id="bar-with-anchors">
```
Comment (Ispirett) [inline index.html:50]: **[Optional] This is not required for your project to be approved but I encourage you to consider.**
- [ ] Try using a  `<nav>` `<ul>` tag and `<li>`

**Review by Ispirett — CHANGES_REQUESTED** [review]
Hey, @nicolasmarino99  great work with the changes requested by @Torres-ssf you are doing a fantastic job :1st_place_medal: . There are just a few key things that need a little addressing

- [x] A round **800px* or so change the width to col-md-12 or col-sm-12 or get the set the width of the itesm to 100%

bstrap


**The orginal site**
bstrap2

- [x]  **html errors**
There are a lot of HTML errors on this page use this tool and please correct them https://validator.w3.org/

Send for another review when finished.

**Review by Ispirett — APPROVED** [review]
Great work with changes made. you can merge your project and submit you complete form :+1: .

**[ Optional] I strongly like advise you to make these changes it will make your project look more professional.**





- [ ] Don't let the text wrap around the images at this resolution and below, around 750px. instead, let the images fill the screen and the text be completely under the images.

- [ ] Also for links it's better to use `<nav> ul and li `tags.

- [ ] Also using article tags instead of divs is a good practice for all the articles listed on the page.

**Pull request merged**